src/frontend/Sider/Behandling/Stønadsvilkår/PassBarn/PassBarn.tsx

```
@@ -25,39 +27,51 @@ const PassBarn: React.FC<Props> = ({ vilkårsregler, vilkårsvurdering }) => {
         (v) => v.vilkårType === Inngangsvilkårtype.PASS_BARN
     );
 
-    return vilkårsvurdering.grunnlag.barn.map((barn) => {
-        const vilkårForDetteBarnet = vilkårsett.filter((e) => e.barnId === barn.barnId);
+    if (vilkårsvurdering.grunnlag['@type'] === Stønadstype.BARNETILSYN) {
```

Hvis det ikke er barnetilsyn her så er det noe som er feil, og kan utgå fra fail fast
```typescript
if (vilkårsvurdering.grunnlag['@type'] !== Stønadstype.BARNETILSYN) {
return <>Feil faktatype for pass av barn</>
}
```
Då trenger man ikke inlinea alt av komponenten i en if, men har kun en sjekk på det rare tilfellet og returnerer noe dersom det er feil

---

Done: 16042406024a3b859fc77ec815ef72ee23e07687